Tasks/JavaToolInstaller/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 0,
         "Minor": 126,
```

Comment:
I think you should use 127.0 since we are in sprint 127.

Reply:
I think Kellie is planning to port to 126, so this is ok.